docs/src/main/sphinx/routines.md

```diff
@@ -1,7 +1,9 @@
 # SQL routines
 
 A SQL routine is a custom, user-defined function authored by a user of Trino in
-a client and written in SQL. More details are available in the following sections:
+a client and written in SQL. Routines are functions that process one or more
```

Comment:
I think this is worse than before. The "one or more input values" makes it sound like an aggregation function. How about

> Routines are scalar functions that return a single output value.

The Trino SPI has `FunctionKind` with `SCALAR` which is what these functions are.

Reply:
If you think of the input arguments as a tuple, then the input is actually a single value. Note that the output type can also be a tuple.

Reply:
ok .. lets go with the scalar version ..

Reply:
Updated now